src/redux/books/books.js

```diff
@@ -0,0 +1,42 @@
+import { apiAddBook, apiGetBooks, apiRemoveBook } from '../../modules/api';
+
+const ADD = 'bookstore/Book/ADD';
+const GET = 'bookstore/Book/GET';
+const REMOVE = 'bookstore/Book/REMOVE';
+
+export const addBook = (payload) => async (dispatch) => {
+  const book = { ...payload, item_id: payload.id, category: 'category' };
+  await apiAddBook(book);
+  dispatch({ type: ADD, book });
+};
+
```

Comment:
![Capture0](https://user-images.githubusercontent.com/93039359/206868746-48846388-6a8a-4fdb-80ac-4da62de213b2.PNG)

- [ ] You have a link of your license file in your readme file 😊 but it is broken 😨 Kindly fix this link to make your readme file descriptive and license file easily accessible to all viewers 👍